designer/server/src/main/resources/db/migration/hsql/V1_036__ProcessActionState.sql

```diff
@@ -0,0 +1,18 @@
+--- DROP CONSTRAINTS
+ALTER TABLE "process_actions" DROP PRIMARY KEY;
+ALTER TABLE "process_actions" ALTER COLUMN "performed_at" SET NULL;
+--- for in-progress actions, version will be null
+ALTER TABLE "process_actions" ALTER COLUMN "process_version_id" SET NULL;
```

Comment:
Because we want to to be able to cancel scenario even if it doesn't have deployed version. It was done this way before my change and it is generally ok - sometimes we have running scenario on flink, but never deployed on our side and we want to be able to cancel it.
The only difference in my change is that, before my change at the end of this successfully finished cancel, it was reported failure in notifications that there is no version and also wasn't added an action - this cancel was done "silently". IMO it wasn't ok, I would like to have this kind of action in the history. Also I need to add in-progress action even in this situation because I need it for changing state of scenario and for blocking other actions purpose. WDYT?